frontend/src/pages/HeadPage.tsx

```diff
@@ -0,0 +1,31 @@
+import {useNavigate} from "react-router-dom";
+
+
+export default function HeadPage(){
+    const navigate = useNavigate()
+
+    function personalDataOnClick(){
+        navigate("/user")
+    }
+
+    function exercisesOnClick(){
+        navigate("/exercises")
+    }
+
+    function weekPlanOnClick(){
+        navigate("/weekplan")
+    }
+
+    function calendarOnClick(){
+        navigate("/calendar")
+    }
```

Comment:
Typischerweise hat man in React die `handle` Syntax

D.h. `handleCalendarNavigation` -/`Click` etc